docs/multi-tenant-db.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Multi-tenant database service
+parent: Matrix Event Bus (MEB)
+grand_parent: SmartClean Matrix
+nav_order: 1
+---
+
+### What is a shared database model
+
+1. This means the same database instance is shared, allowing the following two types of commits:
+   1. You can query data from it. 
+   2. Our system can write data to it. 
+2. Any standard [PostgreSQL](https://www.postgresql.org) database will work. 
+3. The database we use for this is called [TimeScale DB](https://docs.timescale.com) which is provided as an 
+extension to PostgreSQL. 
+4. A multi-tenant component means the component is shared (physically the same, with logical separation)
```

Comment:
A multi-tenant component means the data from projects owned by you is shared with some logical separation and query patterns.

Reply:
Resolved by commit: https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad